experimental/pagerduty-webhook/README.md

```diff
@@ -0,0 +1,41 @@
+### About
+
+Python program to add webhook to the pagerduty triggered and resolved all events, Currently, program only add webhooks and does not replace them. If a webhook with same url exists then it will  add new ones on top. There is still some work need to be done to make sure check existance of webhook and not create the new ones.
+
+### Prerequisites
+
+- Have Python3 installed on machine
+- [Created a personal access token] In Pagerduty go to link https://nandos-chickenland-ltd.pagerduty.com/users/<USER-ID> and create a PagerDuty token by clicking Create API User Token.
+
+### Initialization
+
+Create python virtual environment
+
+```
+python3 -m venv .venv
+```
+
+Activate it:
+
+```
+. .venv/bin/activate
+```
+
+Install into virtual environment
+
+```
+pip install -r requirements.txt
+```
+
+### Running the program
+
+Following is an example of how to add a webhook to a specific repository using `--repository= ` option
+
+```
+python3 program.py --authorizationtoken=mytoken         (from step above) Webhook url (https://dora.nandos.dev) is embedded in the code  
+```
+Above command will write a file which would contain all the services/secrets.
+i.e. 
+PH3DA7T : xxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
Will this be in a format that could be used by Terraform?

Reply:
@matthew-green-nandos  Didn get it..  you mean to save in secret managers? or inject into event handler.